components/Dialogs/examples/DialogsAlertViewController.m

```diff
@@ -47,11 +46,7 @@ - (void)collectionView:(UICollectionView *)collectionView didSelectItemAtIndexPa
   }
 }
 
-
-
 - (IBAction)didTapShowAlert:(id)sender {
-  [[MDCButton appearanceWhenContainedIn:[MDCAlertController class], nil]
```

Comment:
Discussed offline. For the example, the theme will be applied immediately before the alert is presented.